site/content/en/docs/presentations/_index.md

```diff
@@ -105,6 +105,13 @@ Presentations about the minikube project, mostly from the Kubernetes blog and th
   * "Minikube and Three Different Local Kubernetes Learning Environments"
   * Anders Björklund & Predrag Rogic
 
+### KubeCon NA 2021 (Los Angeles, California )
```

Comment:
remove the extra space between California and the )

Reply:
done